cmd/cape/cmd/token.go

```diff
@@ -72,6 +72,11 @@ func token(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	_, err = fmt.Fprintf(cmd.ErrOrStderr(), "This token witll expire in %s\n", time.Second*time.Duration(expiry))
```

Comment:
The logger will print to stderr, so for parity with everything else it'd be better to use something like
```
log.Infof("This token will expire in %s.", time.Second*time.Duration(expiry))
```